src/helpers/audio.helper.ts

```diff
@@ -0,0 +1,66 @@
+import { ICall, StreamMediaType, MediaEvent } from '@/types/rtc'
+
+type ICallKey = keyof ICall
+const CALL_KEYS_TO_INCLUDE: Array<ICallKey> = [
+    'roomId',
+    '_audioMuted',
+    '_cancel_reason',
+    '_contact',
+    'direction',
+    '_end_time',
+    '_eventsCount',
+    '_from_tag',
+    '_id',
+    '_is_canceled',
+    '_is_confirmed',
+    '_late_sdp',
+    '_localHold',
+    '_videoMuted',
+    'status',
+    'start_time',
+    '_remote_identity',
+    'audioTag',
+    //'audioQuality',
+    'isOnHold',
+    //'originalStream',
+    'localMuted'
+]
+
+export function simplifyCallObject (call: ICall): { [key: string]: any } {
+    //const simplified: { [key: string]: ICall[ICallKey] } = {}
+    const simplified: { [key: string]: any } = {} as ICall
+
+    CALL_KEYS_TO_INCLUDE.forEach(key => {
+        if (call[key] !== undefined) {
+            simplified[key] = call[key]
+        }
+    })
+
+    simplified.localHold = call._localHold
+
+    return simplified
+}
+
+export function processAudioVolume (stream: MediaStream, volume: number) {
+    const audioContext = new AudioContext()
+    const audioSource = audioContext.createMediaStreamSource(stream)
+    const audioDestination = audioContext.createMediaStreamDestination()
+    const gainNode = audioContext.createGain()
+    audioSource.connect(gainNode)
+    gainNode.connect(audioDestination)
+    gainNode.gain.value = volume
+
+    return audioDestination.stream
+}
+
+export function syncStream (event: MediaEvent, call: ICall, outputDevice: string, volume: number) {
```

Comment:
In HTML and DOM API, the property to set or get the class of an element is className, not class. This is because class is a reserved keyword in JavaScript. So, you should replace class with className in your interface.

Here's the corrected interface:

```typescript
export interface StreamMediaType extends HTMLAudioElement {
    className: string;
    setSinkId (id: string): Promise<void>;
}
```
And the adjusted syncStream function:
```typescript
export function syncStream (event: MediaEvent, call: ICall, outputDevice: string, volume: number) {
    const audio = document.createElement('audio') as StreamMediaType;

    audio.id = call._id;
    audio.className = 'audioTag';  // use 'className' instead of 'class'
    audio.srcObject = event.stream;
    audio.setSinkId(outputDevice);
    audio.volume = volume;
    
    audio.onloadedmetadata = function() {
        audio.play();
    };

    call.audioTag = audio;
}
```